crates/iroha_executor/src/default/isi/multisig/account.rs

```diff
@@ -0,0 +1,106 @@
+//! Validation and execution logic of instructions for multisig accounts
+
+use iroha_executor_data_model::permission::account::CanRegisterAccount;
+
+use super::*;
+use crate::permission::domain::is_domain_owner;
+
+impl VisitExecute for MultisigRegister {
+    fn visit<V: Execute + Visit + ?Sized>(&self, executor: &mut V) {
```

Comment:
I believe checks in this function are redundant. This will all be verified when calling corresponding `visit_XXX` functions

Reply:
- domain existence ... Yes, can be checked in the execution
- permission to register an account ... Yes, but should be checked before authorized as the domain owner
- signatories existence ... Yes, can be checked in the execution